research/src/pages/tabs.research.parts.mdx

```diff
@@ -0,0 +1,439 @@
+  <figcaption>Carbon design adds next/previous interactive buttons in the horizontal grouped direction (doesn't support a vertical grouped direction)</figcaption>
+</figure>
+
+
+
+<hr />
+
+_Fast does not currently explcitly support anything in particular here, but they are working on it._
+
+
+<hr />
+
+<figure>
+  <img src="https://lh6.googleusercontent.com/GBmQC_vmq6ICMgZzpWjLGF_5oDVZw6Cm7AKKGL9OmsLH3NsJ-gHcRoAQHrLpYYvCTB74W87MwSeEEhj8RIgkZ7waOBSlB3ECXF7xzD29UjmueNIOhSGjqMHZk5SNnRnmqr0T2tLD" />
+  <img src="https://lh4.googleusercontent.com/Bq-34D6YjO7cUH5kq9ev-0ldlae77d4b6wlGWwd1os0Iuf3xFNn8FiIj5CAiiPv12Qi3mtuAx7qmOeDOcXOAdqBKrjqpfgHCw-O99h-4eQlwuk0d0lGw4hvD9ovYbnKn32DiDqYw" />
+  <figcaption>Lightning components adds a trailing interactive "more" dropdown (containing the entire list) in the horizontal grouped direction, and allows both the Tab List and the Tab Panel to take up 100% of the screen, providing a horizontal scroll between them in the vertical grouped direction.
+</figcaption>
+</figure>
+
+
+<hr />
+
+<figure>
+  <img src="https://lh5.googleusercontent.com/5aP7Euq4z4I096U2VM90FSaZsrVg_900MFgJAc1JUgt0UoIP-fIO7EkYALBK0ka4rSXKUrmROjqzRpd8LslY_95AVtFLBRCVS3rzig9xivxvIRI4Mc14hkBNEyOTLv2lSvQY7kWb" />
+  <figcaption>
+Redhat adds interactive next and previous _scroll_ buttons (only handles overflows in horizontal group)
+</figcaption>
+</figure>
+
+
+<hr />
+
+
+<figure>
+  <img src="https://lh5.googleusercontent.com/weukMiWZvOg0lCD2YTGUGjO0dS5vvqgPmV9kjssaWeft7RWk_C3lRemgFlbFmYjEBVzbbECCuXURRhg7yX6ZUmwYDcrQiXOSsZUAWYCQ_KIYhxLemY_r_PW8jxFyJBTJ4T81tSG9" />
+  <figcaption>
+Material clips the Tab List container allows movement of focus or touch scrolling but offers no additional affordances or interactive elements. It only has a horizontal grouped version.
+</figcaption>
+</figure>
+
+
+<hr />
+
+<figure>
+  <img src="https://lh3.googleusercontent.com/z_QukHGXEFyf6yc0N9wl9SjAkida7dgmp1cnw5WlcgmTBDYlSRsqEErlZeug_IoLBMHAC2gXZkqBj9vQQ5vP7muj0SdJlhrVrt79o3yO1rNQnBHQN-K4rHlZQ1qwRzPGWxcb3Ln3" />
+  <img src="https://lh6.googleusercontent.com/t0jeYGxYYMSVVgOCrD4OZbUt6OyvmdLy88isZcifl8XKljv9nzTDD01GlpS9pRJKbDd6VIpgGm4UiAsKbRG6_F2WDmnIa2fLY_P64ey1Z7doy0RC4U_Ptfiy6QCt_BpLfOL5CPeF" />
+  <figcaption>Vaadin overflows the TabList with a vertical scroll bar when grouped vertically, and with interactive next/previous buttons which advance the clipped scroll area by one page when grouped vertically.</figcaption>
+</figure>
+
+
+
+## Visual transitions
+A rarely occurring enhancement, but not entirely uncommon and seemingly growing in popularity is providing a visual transition effect between the finite states of tabs.  It it worth consider a feature on its own in the sense that there are several implementations which are built in such a way that this is not possible to achieve.  
```

Comment:
`It it worth consider` -> `It is worth considering...`

Reply:
updated